include/AJAX_Requests.php

```diff
@@ -0,0 +1,31 @@
+<?php
+include('../include/initialize.php'); 
+
+//I'm planning to send all my AJAX requests to this file from now as I get more familiar with the concept. Is this ok?
```

Comment:
This is alright, but the way we approach it is to have an `ajax` directory, and put a different file for each request. So you might have something like:
- `ajax/get_student.php`
- `ajax/get_class.php`
- 'ajax/login.php`

Since the whole file is going to load, you usually just want to load the code you need for the specific AJAX request you're making. It's not a huge deal to have it this way, but I think you can probably make it a little more organized to have separate files vs. one larger one.